src/main/java/com/rocketden/main/model/problem/ProblemTestCase.java

```diff
@@ -0,0 +1,37 @@
+package com.rocketden.main.model.problem;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+@Entity
+@Getter
+@Setter
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+public class ProblemTestCase {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer id;
```

Comment:
Is there a reason you didn't add the `@EqualsAndHashCode.Include` annotation here?

Reply:
My main thought process being that it's mainly used for the database equality stuff whereas we might want to use a business ID of some sort in our application (similar to in Room how we use userId and nickname)

Reply:
lmk what you think tho, I can add it in for now if you think it's preferable

Reply:
Well I think I agree if we add a problemId, but since we're using the ID for comparisons at least right now, I think we might want to compare by that?

This is just a bit though, and if you think we'll change it in the future I don't have too much of a problem with this.

Reply:
Sounds good - added a commit for this